### ISSUE TITLE
Snap/Flatpack package

### ISSUE THREAD

**Issue by zendre4** [gitahead/gitahead#151]
Hello,

I'm using GitAhead on a Linux desktop, and I'm very with this awesom app. But I think it would be nice to simplify the update process by creating and deploying a Snap package or a Flatpack package.

Personally I use more Snap because It's natively supported in Ubuntu and it's easily configurable on many Linux system , but Flatpack also work well .

Would it be possible for you to implement one of this item ?

Snap :
https://snapcraft.io/ 
https://snapcraft.io/store 

Flatpak :
https://flatpak.org/
https://flathub.org/home

**Comment by hackhaslam** [comment]
The last time we investigated packaging with snap (about a year ago) they didn't have support for some of the distros that we wanted to support, notably RHEL and CentOS. I don't know if the story has changed in the meantime. We can investigate again and take a look at Flatpak as well.

**Comment by FMuro** [comment]
> The last time we investigated packaging with snap (about a year ago) they didn't have support for some of the distros that we wanted to support, notably RHEL and CentOS. I don't know if the story has changed in the meantime. We can investigate again and take a look at Flatpak as well.

Yes, snap has support for those distros through EPEL: https://docs.snapcraft.io/installing-snapd

**Comment by zendre4** [comment]
If you want RHEL ans CentOS, maybe flatpak is better ...

https://flatpak.org/setup/

Apparently Red Hat help the development of Flatpak

**Comment by zendre4** [comment]
It's also possible to install snap on Centos/RHEL:

https://docs.snapcraft.io/installing-snap-on-centos

**Comment by Emilisdls** [comment]
Flatpack seems to be supported [by more distros](https://kamikazow.wordpress.com/2018/06/08/adoption-of-flatpak-vs-snap-2018-edition/)

**Comment by Croydon** [comment]
👍 for a Flatpak version

**Comment by zendre4** [comment]
Recently I make some Qt app for my enterprise in Flatpak... It's very simple to generate an Flatpak app with the KDE runtime/SDK, because a lot of job was realise by the KDE team for integrate Qt and is dependencies, and also keep an up-to date version of Qt.

I also noticed that the application catalogue has been growing a lot lately. So flatpak is a very interesting solution.

**Comment by alculete** [comment]
> It's also possible to install snap on Centos/RHEL:
> 
> https://docs.snapcraft.io/installing-snap-on-centos
+1 for snap

**Comment by spipau** [comment]
I just tired to install GitAhead with the .sh file and failed on Ubuntu 20.04.2 - I guess I could fix it by investigating, but I simply do not want to waste time on installing software. Please just properly pack your software and make it easily installable + updatable. Thanks for your hard work :star:

**Comment by Croydon** [comment]
@advaithm and @barthalion created a Flatpak version 🥳  

https://flathub.org/apps/details/io.github.gitahead.GitAhead

https://github.com/flathub/io.github.gitahead.GitAhead

**Comment by safield** [comment]
As an Ubuntu user, I would like to see snap support as well. As snap and flatpak are in a land grab stage with quite an equal distribution between users, it would probably be best to support both.....if possible.
